### ISSUE TITLE
Broken on pyelftools 0.27

### ISSUE THREAD

**Issue by implr** [Vector35/dwarf_import#9]
Error on import:
```
Failed to import python plugin: official/Vector35_dwarf_import: cannot import name 'GenericExprVisitor' from 'elftools.dwarf.dwarf_expr' (/usr/lib/python3.8/site-packages/elftools/dwarf/dwarf_expr.py)
```
Downgrading to 0.26 fixes this.

**Comment by TACIXAT** [comment]
Also running into this issue. `GenericExprVisitor` was removed March of 2020. [Relevant commit.](https://github.com/eliben/pyelftools/commit/4cf5cb07c772002ee736dba17fc685a260817b7a)

**Comment by TACIXAT** [comment]
`pip install -Iv pyelftools==v0.26`

**Comment by KyleMiles** [comment]
Resolved by #8.
